tests/test_file.py

```diff
@@ -204,9 +204,15 @@ def _test_file_get_not_found(self, replica):
                 requests.codes.not_found
             )
 
+            self.assertHeaders(
+                response[0],
+                {
+                    'content-type': "application/problem+json",
+                }
+            )
             self.assertEqual(response[2]['code'], "not_found")
```

Comment:
nit-picky, but I think it's cleaner to say `response, data, actual_json = AssertGetResponse(...)` and then use `actual_json` instead of `response[2]`. Thoughts?